testsuite/tests/shapes/recmodules.ml

```diff
@@ -79,20 +75,19 @@ and ASet : sig
 end = Set.Make(A)
 [%%expect{|
 {
- ("A", module) -> {
-                   ("compare", value) -> <.38>;
-                   ("t", type) -> <.35>;
-                   };
- ("ASet", module) ->
+ "A"[module] -> {
+                 "compare"[value] -> <.38>;
+                 "t"[type] -> <.35>;
+                 };
+ "ASet"[module] ->
      {
-      ("compare", value) ->
-          CU Stdlib . "Set"[module] . "Make"[module](A/324<.19>) .
+      "compare"[value] ->
+          CU Stdlib . "Set"[module] . "Make"[module](<.19>) .
```

Comment:
Looks like the functor argument (`A`, defined in the same recursive group) which was previously a `Var` is now a `Leaf`.

Reply:
(fixed)